README.md

```diff
@@ -27,6 +27,13 @@ Lag en `.env` fil i `src/backend` og legg inn:
 AZURE_APP_CLIENT_ID=<clientId>
 AZURE_APP_CLIENT_SECRET=<clientSecret>
 ```
+# Feilsøking
+Feilmelding ved oppstart av app: 
+```
+Error [ERR_MODULE_NOT_FOUND]: Cannot find module '<...>/src/backend/build/auth/attachToken' imported from <...>/src/backend/build/server.js 
+```
+Løsning: Du trenger Node-versjon 18. 
+
```

Comment:
Burde vært gjort utenfor denne PR-en 👀 

Stemmer dette nå egentlig? Jeg fikk en feilmelding i går om feil versjon av 18 også, men så ut som jeg enten kunne ha 18.noe eller mer enn 20.noe 😕

Reply:
Jeg nedgraderte til node 18 fra 20, og da fungerte det igjen. 

Du tenker nok på en annen feil, nemlig at `eslint-plugin@7.4.0` krever node-versjon `"^18.18.0 || >=20.0.0"`

> Burde vært gjort utenfor denne PR-en 👀

Du må gjerne forklare hva du tenker. Ser selv ikke problemet. 

Alle helst skulle jeg ha pusha dette rett i main. Syntes ikke vi skal gjøre det tyngre enn nødvendig for oss selv å gjøre små forbedringer, selv om det ikke direkte har noe med innholdet i PR-en å gjøre forøvrig. Det blir noe annet hvis PR-en allerede er stor, men det kan vi ikke si her akkurat.